src/main/java/tr/com/orioninc/laborant/service/LabService.java

```diff
@@ -42,13 +55,14 @@ public static String connectAndExecuteCommand(String username, String password,
             channel.connect();
 
             while (channel.isConnected()) {
-                Thread.sleep(500);
+                Thread.sleep(200);
             }
 
             responseString = new String(responseStream.toByteArray());
         } catch (JSchException e) {
             log.error("[connectAndExecuteCommand] ssh exception: {}", e.getMessage(), e);
-            return null;
+            responseString = "SSH Exception: " + e.getMessage();
+            return "SSH Exception: " + e.getMessage();
```

Comment:
Don't return the exception to the cleint

Reply:
done

Reply:
Client still gets the exception with `e.getMessage()`

Reply:
sorry for that, I commit again